sdk/core/core-http/review/core-http.api.md

```diff
@@ -181,7 +181,7 @@ export class DefaultHttpClient extends FetchHttpClient {
     }
 
 // @public
-export function delay<T>(t: number, value?: T): Promise<T | void>;
+export function delay<T>(delayInMs: number, abortSignal?: AbortSignalLike, abortErrorMsg?: string, value?: T): Promise<T | void>;
```

Comment:
`value?: T` should be the 2nd argument, otherwise this would be a breaking change.

Reply:
Good point.

I'll switch to that.
What about the core-amqp counterpart? How to get the copy of this method to stay consistent?

Reply:
@HarshaNalluru I think there is no need to care about their consistency, we can just ignore them once `core-util` goes GA. Also, `core-http` is going to be phased out fairly soon anyway.